tests/dev_env_tests/test_local_global_step_logic.py

```diff
@@ -29,26 +29,14 @@ def get_parametrization_list_success_cases():
             },
         },
     }
-
-    expected_response = {
-        "title": "Standard Deviation",
-        "columns": [
-            {"name": "variable", "type": "STR", "data": ["lefthippocampus"]},
-            {"name": "std_deviation", "type": "FLOAT", "data": [0.3773485998788057]},
-        ],
-    }
-
-    parametrization_list = []
-    parametrization_list.append((algorithm_name, request_dict, expected_response))
-
-    return parametrization_list
+    return [(algorithm_name, request_dict)]
```

Comment:
I think that this test could also assert the result of the algorithm, just not the data.

So this test could remain the same, without the last 4 lines asserting the data.

```
    # Assert the standard deviation with 4 decimals only
    assert round(float(response.json()["columns"][1]["data"][0]), 4) == round(
        float(expected_response["columns"][1]["data"][0]), 4
    )
```

It's not bad to test that the result format is the expected one.

What do you think?

Reply:
This test was introduced to test the new local/global step wrapper. It should only test that an algorithm using this feature is able to run beginning to end, i.e. `status_code=200`. Why should it also test that `TabularDataResult` returns the expected format? This is or should be tested elsewhere.

Reply:
My only concern is that the algorithm succeeds but for some reason the result is not the expected.
This could mean that the algorithm didn't actually work as expected. If for example a "secure_transfer" object is not aggregated properly from the "merge_secure_transfer".

That's the initial reason that I was also testing the data and maybe we should keep it like it was. I was not testing that the response is numerically correct but that the response will not change, if something changes in the system.

Reply:
The algorithm in question (standard deviation) doesn't use `secure_transfer` and `merge_secure_transfer`. Am I missing something?

It uses `transfer` and `merge_transfer`. If these don't work as expected (but the algorithm still returns 200) the numerical results will be affected, not the structure of the result DTO. So it doesn't make sense to me to test for the result DTO structure but not for the numerical results, if our goal is to test `transfer` etc.

On the other hand the above functions are used in PCA. This PR introduces PCA +100 validation tests, where the above features are used and where everything is tested, i.e. the successful execution, the result format, the result values, etc.

Reply:
Yeah sorry, I meant `transfer` and `merge_transfer`...

Ok I see your point about PCA testing those anyway. I was thinking that standard deviation is testing mainly these things so if it fails you know more specifically that the problem is there.

If you think that this doesn't add much value, no problem. Up to you.